core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -13,7 +13,6 @@
 <script src="/third_party/static/headroom-js-0.9.4/headroom.min.js"></script>
 <script src="/third_party/static/headroom-js-0.9.4/angular.headroom.min.js"></script>
 <script src="/third_party/static/angular-drag-and-drop-lists-2.1.0/angular-drag-and-drop-lists.min.js"></script>
-<script src="/third_party/static/angular-recorder-1.4.1/dist/angular-audio-recorder.min.js"></script>
```

Comment:
Mistake delete?

Reply:
No it was deleted in another PR so when resolving merge conflicts, I had to delete it